x/oracle/keeper/tally.go

```diff
@@ -25,8 +26,8 @@ func (k Keeper) BuildClaimsMapAndTally(ctx sdk.Context, params types.Params) err
 
 	var (
 		// voteTargets defines the symbol (ticker) denoms that we require votes on
-		voteTargets      []string
-		voteTargetDenoms []string
+		voteTargets      = make([]string, len(params.AcceptList))
+		voteTargetDenoms = make([]string, len(params.AcceptList))
```

Comment:
And this is why I hate using `make([]blah, N)` instead of `make([]blah, 0, N)`, 
as you can see, the code below uses append and when you append to a slice initialized to length instead of capacity, it will panic somewhere else or produce weird bugs, because some elements will be zero..

<img width="1707" alt="Screenshot 2023-03-07 at 17 59 01" src="https://user-images.githubusercontent.com/477998/223494810-08af1b9d-91cb-4c1e-ab24-619279f4c21c.png">

Reply:
I think it's fine to do it `make([]string, 0, len())`